src/argocd.ts

```diff
@@ -0,0 +1,20 @@
+// argocd.ts - ArgoCD Components
+import { App } from './interfaces/App';
+
+export function filterAppsByExcludedPath(apps: App[], excludedPaths: string[]): App[]{
+  return apps.filter(app => {
+    return !excludedPaths.includes(app.spec.source.path);
+  });
+}
```

Comment:
```suggestion
export function filterAppsByExcludedPath(apps: App[], excludedPaths: string[]): App[]{
  if (excludedPaths.length === 0) {
    return apps;
  }
  
  return apps.filter(app => {
    return !excludedPaths.includes(app.spec.source.path);
  });
}
```

If we're concerned with speed, we should move the if statement into here. Though it shouldn't alter the correctness.

I've also updated the if statement to check array length, there might be a better way to check an empty array in typescript but I'm unsure.